source/Scenes/World/WorldScene.cs

```diff
@@ -6,20 +6,21 @@
 using Game.Entities;
 using Game.Events;
 using System.Collections.Generic;
+using Game.Entities.Planets;
 
 namespace Game.Scenes.World
 {
     public class WorldScene : AstroSoarScene, ISceneWithPlayer
     {
-        private readonly IList<BaseEntity> _entities;
-        public Player? Player { get; set; }
+        private readonly IList<Entity> _entities;
+        public Player Player { get; set; }
 
         public WorldScene()
         {
-            _entities = new List<BaseEntity>()
+            Player = new Player();
```

Comment:
Player should be initialized already through the ISceneWithPlayer interface and shouldn't be directly initialized here.

Reply:
Check out HandleSceneOnEnter in the AstroSoarScene for how the player gets loaded